tests/components/cloud/test_account_link.py

```diff
@@ -96,6 +96,18 @@ async def test_get_services_cached(hass):
         assert await account_link._get_services(hass) == 4
 
 
+async def test_get_services_error(hass):
+    """Test that we cache services."""
+    hass.data["cloud"] = None
+
+    with patch.object(account_link, "CACHE_TIMEOUT", 0), patch(
+        "hass_nabucasa.account_link.async_fetch_available_services",
+        side_effect=asyncio.TimeoutError,
+    ):
+        assert await account_link._get_services(hass) == []
+        assert account_link.DATA_SERVICES not in hass.data
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5518609)